### DISCUSSION TITLE
nvim Error

### DISCUSSION THREAD

**Discussion by TheCollinsByte** [2KAbhishek/dots2k#14] [Q&A]
I am getting:
`Error detected while processing /home/{username}/Dotfile/.config/nvim/init.lua`
`vim:E6100: "state" is not a valid stdpath`

**Comment by TheCollinsByte** [comment]
I cloned your `nvim2k` but I am having a problem in starting the nvim.

**Comment by TheCollinsByte** [comment]
It's not able to download the plugins.

**Comment by TheCollinsByte** [comment]
When I try to run `:UpdateRemotePlugins`.
 I get `remote/host: generated rplugin manifest: ~/.local/share/nvim/rplugin.vim`

**Comment by 2KAbhishek** [comment]
You need to use :Lazy install for installing plugins

**Reply by TheCollinsByte** [reply]
I get `E492: Not an editor command: Lazy` when I run `:Lazy`

**Reply by 2KAbhishek** [reply]
Can you please check lazy's readme for installation instructions
